Firestore/core/src/firebase/firestore/model/transform_operation.h

```diff
@@ -193,6 +199,13 @@ class NumericIncrementTransform : public TransformOperation {
  public:
   explicit NumericIncrementTransform(FieldValue operand);
 
+  /**
+   * Casts a TransformOperation to a NumericIncrementTransform. This is a
+   * checked operation that will assert if the type of the TransformOperation
+   * isn't actually Type::Increment.
```

Comment:
I was going to complain that I don't see this assert.  But actually I don't see this constructor implemented at all.

Reply:
Apologies--I'm going too fast and Travis doesn't check PRs stacked on PRs. Fixed.